scss/components/_charts.scss

```diff
@@ -0,0 +1,313 @@
+// the amount of padding at the bottom of vertical bar charts
+$chart-bottom: 30px;
+// the width of the axis for vertical bar charts
+$chart-axis-width: 120px;
+
+$tooltip-arrow-size: 8px;
+$tooltip-border-color: #999;
+
+.chart-container {
+  font-family: $sans-serif;
+  border: 1px solid $neutral;
+
+  .chart-series {
+    margin: 1rem;
+  }
+
+  & + .chart-container {
+    margin-top: 4rem;
+  }
+}
+
+.chart-series__bar {
+  background: $primary;
+}
+
+.data--disbursement,
+.data--disbursements,
+.data--total_disbursements_period,
+.data--independent_contributions_period {
+  background: $primary;
+  .chart-series__bar__tooltip {
+    color: $primary;
+  }
+}
+
+.data--receipts,
+.data--total_receipts_period,
+.data--independent_expenditures_period {
+  background: $primary-contrast;
+  .chart-series__bar__tooltip {
+    color: $primary-contrast;
+  }
+}
+
+.data--cash,
+.data--cash_on_hand_end_period {
+  background: $primary-contrast;
+  .chart-series__bar__tooltip {
+    color: $primary-contrast;
+  }
+
+}
+
+.data--debt,
+.data--debts_owed_by_committee {
+  background: $primary;
+  .chart-series__bar__tooltip {
+    color: $primary;
+  }
+}
+
+.data--support { background: $primary; }
+.data--oppose { background: $primary-contrast; }
+
+.chart-series {
+  position: relative;
+  margin: 1em 0;
+}
+
+// horizontal bar charts
+.chart-series--horizontal {
+
+  .chart-series__bar {
+    padding: 1px;
+    white-space: nowrap;
+    min-height: 1.5em;
+  }
+
+  .chart-series__group {
+    margin-bottom: 2em;
+  }
+
+  .chart-series__axis {
+    display: none;
+  }
+}
+
+// vertical bar charts use flexbox
+// see: <http://caniuse.com/#feat=flexbox>
+.chart-series--vertical {
+  @include display(flex);
+  @include flex-direction(row);
+  @include align-items(flex-end);
+  padding-bottom: $chart-bottom;
+  padding-left: 0;
+  height: 200px;
+
+  @include media($med) {
+    padding-left: $chart-axis-width;
+  }
+
+  &.chart-series--short {
+    height: 100px;
+  }
+
+  .chart-series__bar,
+  .chart-series__group {
+    @include flex(1);
+
+    // for positioning labels
+    padding-top: 1px;
+    position: relative;
+    height: 100%;
+
+    margin-left: 10%;
+
+    &:first-of-type {
+      margin-left: 0;
+    }
+  }
+
+  .chart-series__bar {
+    // XXX
+  }
+
+  .chart-series__group {
+    @include display(flex);
+    @include flex-direction(row);
+    @include align-items(flex-end);
+    max-width: 90px;
+    z-index: 9;
+
+    .chart-series__bar + .chart-series__bar {
+      margin-left: .5em;
+    }
+  }
+
+  .chart-series__axis {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: $chart-bottom;
+
+    @include media($med) {
+      left: $chart-axis-width;
+      right: 20px;
+    }
+  }
+
+  .chart-series__axis__tick {
+    position: absolute;
+    text-align: right;
+    width: $chart-axis-width;
+    left: 0;
+    width: 100%;
+    height: 1em;
+
+    .chart-series__axis__tick__text {
```

Comment:
I think this is too many levels of elements. Maybe something like this:

```
.chart-tick
.chart-tick__text
```

No need to fix now.